pkgs/by-name/bs/bs-manager/package.nix

```
@@ -0,0 +1,173 @@
+{
+  alsa-lib,
+  autoPatchelfHook,
+  atk,
+  bash,
+  cairo,
+  cups,
+  dbus,
+  dpkg,
+  fetchurl,
+  ffmpeg,
+  glib,
+  glibc,
+  gtk3,
+  lib,
+  libappindicator-gtk3,
+  libdrm,
+  libxkbfile,
+  libxkbcommon,
+  makeWrapper,
+  mesa,
+  nss,
+  nspr,
+  nix-update-script,
+  pango,
+  pkgs,
+  python3,
+  opencomposite,
+  stdenv,
+  systemd,
+  vulkan-loader,
+  xorg,
+  depotdownloader,
+  openssl,
+  wrapGAppsHook3,
+}:
+let
+  version = "1.5.0-alpha.8";
+
+  src =
+    if stdenv.hostPlatform.system == "x86_64-linux" then
+      fetchurl {
+        url = "https://github.com/Zagrios/bs-manager/releases/download/v1.5.0-alpha.8/bs-manager_1.5.0-alpha.8_amd64.deb";
+        hash = "sha256-xQ3ha6J1wqzZj25QwlMX2lwDDNXlzVvpUqXkaH9kCbE=";
+      }
+    else
+      throw "BSManager is not available for your platform";
+
+  rpath = lib.makeLibraryPath [
+    alsa-lib
+    atk
+    cairo
+    cups
+    dbus
+    ffmpeg
+    glib
+    glibc
+    gtk3
+    libappindicator-gtk3
+    libdrm
+    libxkbcommon
+    libxkbfile
+    mesa
+    nspr
+    nss
+    pango
+    stdenv.cc.cc
+    vulkan-loader
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libXScrnSaver
+    xorg.libXtst
+    xorg.libxcb
+    xorg.libxkbfile
+  ];
+in
+stdenv.mkDerivation {
+  pname = "bs-manager";
+  inherit version;
+
+  system = "x86_64-linux";
+
+  inherit src;
+
+  runtimeDependencies = [
+    bash
+    opencomposite
+    python3
+    systemd
+    depotdownloader
+    openssl
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    ffmpeg
+    gtk3
+    mesa
+    nspr
+    nss
+    makeWrapper
+    openssl.dev
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    dpkg
+    ffmpeg
+    openssl.dev
```

```suggestion
    ffmpeg
    openssl
```
.dev is the default.
Also I don't really understand why gtk3 mesa etc. are in nativeBuildInputs. They are runtime dependencies, right?

---

I don't actually remember why, that must have been added during my sleep deprived state of trying to get this working

---

ah now I know, mesa is needed for some mesa libraries during the build. Removing mesa from the `nativebuildinputs` prevents the application from building